src/components/Game.js

```diff
@@ -0,0 +1,66 @@
+import { useMemo, useState } from "react";
+import Board from "./Board";
+
+function Game(props) {
+  const resetbox = () => {
+    return Array(9).fill("");
+  };
+  const [xIsNext, setxIsNext] = useState(true);
+  const [box, setBox] = useState(resetbox);
+
+  const handleClick = (i) => {
+    if (calculateWinner(box) || box[i]) {
+      return;
+    }
+    box[i] = xIsNext ? "X" : "O";
+    setBox(box);
+    setxIsNext(!xIsNext);
+  };
+
+  const winner = useMemo(() => {
+    let turn;
```

Comment:
you can define const here so that you don't have to re-calculateWinner every time you use 

`const winner = calculateWinner(boxes)`